src/feedrattler/utils.py

```diff
@@ -203,3 +203,23 @@ def remove_empty_script_test(yaml_file_path: os.PathLike):
 
     with open(yaml_file_path, "w") as f:
         yaml.dump(data, f)
+
+def rename_bld_bat_to_build_bat(yaml_file_path: os.PathLike):
+    """If present, rename bld.bat to build.bat"""
+
+    yaml_file_folder = os.path.dirname(yaml_file_path)
+    candidate_bld_bat_path = os.path.join(yaml_file_folder, "bld.bat")
+
+    if not os.path.exists(candidate_bld_bat_path):
+        return
+
+    build_bat_path = os.path.join(yaml_file_folder, "build.bat")
+
+    # If bld.bat exists but build.bat already exits, do not rename and print a warning
+    if os.path.exists(build_bat_path):
+        logging.warning("⚠️ `bld.bat` script exists, but `build.bat` script already exists so bld.bat is not renamed in build.bat")
+        return
```

Comment:
Do you think it would be best to raise an exception in that case? That seems like an odd situation no?

Reply:
Good point, I can change it.